packages/xo-server/config.toml

```diff
@@ -92,3 +92,6 @@ timeout = 600e3
 
 # see https:#github.com/vatesfr/xen-orchestra/issues/3419
 # useSudo = false
+
+# Maximum @xen-orchestra/fs parallel operations
```

Comment:
This does not bring any new info than those in the name.

Also no need to explicit `@xen-orchestra/fs`, all options in this section are for it.